DevHome.sln

```diff
@@ -58,20 +58,12 @@ EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "DevHome.Settings", "settings\DevHome.Settings\DevHome.Settings.csproj", "{72CE876A-7272-4460-92EB-845A45B743FB}"
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "DevHome.SetupFlow.ElevatedServer", "tools\SetupFlow\DevHome.SetupFlow.ElevatedServer\DevHome.SetupFlow.ElevatedServer.csproj", "{940FD524-1AC0-4BBA-BBBE-1E4F2E797508}"
-	ProjectSection(ProjectDependencies) = postProject
-		{A7E5FD7B-B41A-4CAE-A45A-E686DFA8ACF1} = {A7E5FD7B-B41A-4CAE-A45A-E686DFA8ACF1}
-	EndProjectSection
-EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "DevHome.SetupFlow.ElevatedComponent", "tools\SetupFlow\DevHome.SetupFlow.ElevatedComponent\DevHome.SetupFlow.ElevatedComponent.csproj", "{A7E5FD7B-B41A-4CAE-A45A-E686DFA8ACF1}"
-EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "DevHome.SetupFlow.ElevatedComponent.Projection", "tools\SetupFlow\DevHome.SetupFlow.ElevatedComponent.Projection\DevHome.SetupFlow.ElevatedComponent.Projection.csproj", "{053AF75C-5CD8-497F-BA25-47435BD86047}"
-	ProjectSection(ProjectDependencies) = postProject
-		{A7E5FD7B-B41A-4CAE-A45A-E686DFA8ACF1} = {A7E5FD7B-B41A-4CAE-A45A-E686DFA8ACF1}
-	EndProjectSection
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "CoreWidgetProvider", "CoreWidgetProvider", "{32C0052F-10AF-48C0-A7A3-B0A1793397FD}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "CoreWidgetProvider", "CoreWidgetProvider\CoreWidgetProvider.csproj", "{0D879E08-99AA-4019-9D04-DEA9F7C7BFC1}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "CoreWidgetProvider", "CoreWidgetProvider\CoreWidgetProvider.csproj", "{0D879E08-99AA-4019-9D04-DEA9F7C7BFC1}"
+EndProject
+Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "DevHome.SetupFlow.Contract", "tools\SetupFlow\DevHome.SetupFlow.Contract\DevHome.SetupFlow.Contract.vcxproj", "{493E025E-AA0A-44A2-A2A5-6DA88329194C}"

```

Comment:
Nit: I'm not a fan of the name `DevHome.SetupFlow.Contract`. `Contract` seems too generic for how specific it is; maybe something like `TaskResultContract`?

Reply:
shouldn't this be part of the SDK? why a separate project?
cc @EricJohnson327

Reply:
I don't think it needs to be part of the SDK unless we want to allow extensions that extend the setup flow. And that would be an item bigger than this change.